lib/index.js

```diff
@@ -97,7 +97,7 @@ exports.parseCOinSTitle = BBPromise.method(function(title){
 		element = element.split('=');
 		if (element.length!== 2){return;} // Invalid element
 		key = element[0];
-		value = decodeURIComponent(element[1]);
+		value = decodeURIComponent(element[1]).replace(/\+/g,' '); // Replace + since they aren't getting decoded as spaces
```

Comment:
If you do it like this, you would incorrectly decode `%2B` to space instead of `+`. I propose to do the replace before `decodeURIComponent`. If it's possible you have `%2B` in the input.